malboxes/malboxes.py

```diff
@@ -487,6 +487,11 @@ def prepare_profile(template, config):
         for package_mod in profile["package"]:
             package(profile_name, package_mod["package"], fd)
 
+    if "packer" in profile:
```

Comment:
Do you plan to add other fields to the `packer` attribute? For example, If there's only `provisionners`, I'd delete `packer` and make it clearer IMO. Also, if you plan to add other fields other than `provisionners`, what would they be?

Reply:
I kept in mind the possibility because I might plan on adding support for post-processors (mostly shell like this in order to allow some workflow processinng after the box generation.

Reply:
I kept in mind the possibility because I might plan on adding support for post-processors (mostly shell[1] like this in order to allow some workflow processinng after the box generation.

[1]: https://www.packer.io/docs/post-processors/shell-local.html

Reply:
Cool!

Reply:
Wait, I need to fix a missing comma in default-profile.js, right under comment, I on the train right now.